SourceX/miniwin/misc_msg.cpp

```diff
@@ -62,7 +62,7 @@ static int translate_sdl_key(SDL_Keysym key)
 		return DVL_VK_OEM_7;
 	case SDLK_MINUS:
 		return DVL_VK_OEM_MINUS;
-	case SDLK_PLUS:
+	case SDLK_EQUALS:
```

Comment:
IMO this should be also `SDLK_KP_PLUS` as in "classic" diablo you can zoom in/out using +/- also on keypad.